pkg/agent/types/networkpolicy.go

```diff
@@ -77,3 +77,13 @@ func (p *Priority) Less(p2 Priority) bool {
 	}
 	return p.TierPriority > p2.TierPriority
 }
+
+type RuleMetric struct {
+	Bytes, Packets, Sessions uint32
```

Comment:
uint32 is easy for Bytes to overflow, it should use uint64?

Reply:
Make sense, updated.